packages/block-library/src/query/edit/inspector-controls/index.js

```diff
@@ -52,6 +53,7 @@ export default function QueryInspectorControls( props ) {
 		inherit,
 		taxQuery,
 		parents,
+		postFormat,
```

Comment:
We also need to update `onPostTypeChange` below, to update the applied formats, if any..

Reply:
Fixed to the best of my ability, it seems to work.